app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -32,8 +32,8 @@ def sync_nexus_regions
       end
 
       def backfill_transactions
-        @backfilled_order_numbers = ::SuperGood::SolidusTaxjar::BackfillTransactions.new.call
-        flash[:success] = "Successfully backfilled transactions for #{@backfilled_order_numbers.count} orders."
+        @transaction_sync_batch = ::SuperGood::SolidusTaxjar::BackfillTransactions.new.call
+        flash[:success] = "Successfully backfilled transactions for #{@transaction_sync_batch.transaction_sync_logs.count} orders."
```

Comment:
Just wanted to note that since now we are actually handling failures to report transactions, it's possible this message will be incorrect as not all of the logs may correspond to successful transactions. Totally fine if we want to handle that in a future PR!

Reply:
Thats a good callout @Noah-Silvera! We can easily split this up based on the status to errored/succeeded counts. As you mentioned we're already working on the batching UI, so we'll have another PR shortly to redirect to the new views where the user can see the detailed logs.

Reply:
We changed the copy here in preparation for moving this to an async job in a future PR. Let me know if this makes more sense in terms of the language.